utils/lewis_launcher.py

```diff
@@ -185,27 +200,49 @@ def backdoor_set_on_device(self, variable_name, value):
         else:
             self.backdoor_command(["device", str(variable_name), str(value)])
 
+    def backdoor_run_function_on_device(self, function_name, arguments=None):
+        """
+        Run a function in lewis using the back door on a device.
+
+        :param function_name: name of th function to call
+        :param arguments: an iterable of the arguments for the function; None means no arguments. Arguments will
+            automatically be turned into json
+        :return:
+        """
+        command = ["device", function_name]
+        if arguments is not None:
+            for argument in arguments:
+                if isinstance(argument, str):
+                    command.append("'{0}'".format(argument))
```

Comment:
Minor note, the `0` doesn't add any value here. `"'{}'".format(argument)` would work without any loss of clarity